api/src/migrations/20210315122922_remove_sale_offer_event_table.ts

```diff
@@ -0,0 +1,10 @@
+import * as Knex from "knex";
+
+
+export async function up(knex: Knex): Promise<void> {
+}
+
+
+export async function down(knex: Knex): Promise<void> {
+}
```

Comment:
The migrations seem a bit off. Are all of these needed? This one's empty for example

In short, what needs to be migrated, and can that migration be done in a single migration file?